syntax = "proto3";

package api;

// Route /user/login
message LoginRequest {
	string username = 1;
	string password = 2;
}

// Route /user/logout
message LogoutRequest {
	int64 user_id = 1;
}

// Route /user/request-id-range
message IdRangeResponse {
	int64 start = 1;
	int64 end = 2;
}
